pkg/kapis/openpitrix/v1/handler.go

```diff
@@ -927,6 +927,11 @@ func handleOpenpitrixError(resp *restful.Response, err error) {
 		api.HandleBadRequest(resp, nil, err)
 		return
 	}
+	if status.Code(err) == codes.FailedPrecondition {
```

Comment:
to make it simpler, could squash this and the upper if into one statement
```suggestion
	if status.Code(err) == codes.InvalidArgument || status.Code(err) == codes.FailedPrecondition {
```